test/helpers/utils.spec.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-env mocha */
+import { assert } from 'chai';
+import UtilsHelper from '../../helpers/utils';
+import config from '../../config/config.json';
+
+describe('helpers/authorization', () => {
+  it('should return a correct authorization url to call the API "/api/v1/authorize"', () => {
+    // Setup
```

Comment:
I rather remove this comments as we won't repeat this on every tests made since it will weighten the reading without adding enough useful information